Nodejs/Product/Nodejs/Templates/Files/TypeScriptTsConfig/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "compilerOptions": {
+    "noImplicitAny": false,
```

Comment:
ok, do you think it's  good idea to include "compile on save" by default as well?